### ISSUE TITLE
Dashboard is not exported when download all configurations using version 1.15.0

### ISSUE THREAD

**Issue by cyang1001** [dynatrace-oss/terraform-provider-dynatrace#175]
**Describe the bug**
There are no dashboard configurations downloaded when trying to download all configurations using provider version 1.15.0 and `-export` command. 

**To Reproduce**

Follow the instructions for exporting:
1. assign the values of DT_ENV_URL and DT_API_TOKEN in command line.
2. run `terraform init` with provider version "1.15.0" to obtain the provider executable
3. run `.terraform\providers\registry.terraform.io\dynatrace-oss\dynatrace\1.15.0\windows_amd64\terraform-provider-dynatrace_v1.15.0.exe -export`

Scopes being used to download the configurations: 

- Read configuration
- Read SLO
- Read settings
- Read synthetic locations 
- Read synthetic monitor executions 
- Read synthetic monitors, locations, and nodes
- Access problem and event feed, metrics, and topology
- Dynatrace module integration - Davis
- Read credential vault entries

To grab the configurations for the dashboards, I need to run `.terraform\providers\registry.terraform.io\dynatrace-oss\dynatrace\1.15.0\windows_amd64\terraform-provider-dynatrace_v1.15.0.exe -export dynatrace_dashboard` explicitly, or use the legacy export method i.e. `exportv1`.

**Expected behavior**
Dashboard configurations are downloaded along with others when running `.terraform\providers\registry.terraform.io\dynatrace-oss\dynatrace\1.15.0\windows_amd64\terraform-provider-dynatrace_v1.15.0.exe -export`.

**Screenshots**
![image](https://user-images.githubusercontent.com/122544857/212100415-bd3e1e92-eed2-4ce0-b3da-7966c3b87841.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 20H2
 
Dynatrace cluster version: 1.254

**Comment by kishikawa12** [comment]
Hi @cyang1001,

That is expected behavior - as noted in the [Export Utility](https://registry.terraform.io/providers/dynatrace-oss/dynatrace/latest/docs/guides/export-v2) documentation. Due to the number of dashboards that could be in a tenant, we decided to exclude dashboard export from the download all. What you mentioned of explicitly specifying it in the command line arguments is the correct usage.

We found that certain environments could have thousands of dashboards which could significantly lengthen the amount of time to download all configuration. In order to provide a choice for users, we decided to leave it out of the default download all but to allow to explicitly pull the configuration.

I apologize for any inconvenience, we'll continue to evaluate how we can make it easier to utilize the export utility.

Regards,
Kodai
